app/pages/home.cjsx

```diff
@@ -79,7 +79,7 @@ module.exports = React.createClass
     document.documentElement.classList.remove 'on-home-page'
 
   lastFourProjects: ->
-    @props.user.get("project_preferences", page_size: 4, sort: '+updated_at')
+    @props.user.get("project_preferences", page_size: 4, sort: 'updated_at')
```

Comment:
Indeed - i should look at the results that came back...not just the sql scope! Updated.